packages/iTwinUI-react/src/core/Dialog/DialogContext.tsx

```diff
@@ -51,7 +51,7 @@ export type DialogContextProps = {
    * Whether dialog should be positioned relatively to a container or the viewport.
    *
    * Using `'container'` will absolutely position this dialog relative to the closest positioned ancestor.
-   * In other words, you must place the dialog as a child of an element that has `position` set to 
```

Comment:
Could you undo this?

Reply:
**[DONE]** Ohh, I didn't know I made that change. Looks like it was the formatter that did this. Undid it now